dist/build/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM clux/muslrust:1.30.0-stable
```

Comment:
in previous PR you wanted newer image maybe it worth to use newer here?

Reply:
The version bump is in #25. I wanted to split it apart but it looks like the upstream image changes make it difficult.